.github/scripts/execute_single_benchmark.sh

```diff
@@ -15,13 +15,18 @@ echo "Config: ${CONFIG_NAME}, Dataset: ${DATASET_NAME}, Download Source: ${DATAS
 # Prepare the download URLs.
 if [ "$DATASET_NAME" == "skydio-8" ]; then
   export GDRIVE_FILEID='1mmM1p_NpL7-pnf3iHWeWVKpsm1pcBoD5'
+
 elif [ "$DATASET_NAME" == "skydio-32" ]; then
   export GDRIVE_FILEID='1BQ6jp0DD3D9yhTnrDoEddzlMYT0RRH68'
+
 elif [ "$DATASET_NAME" == "palace-fine-arts-281" ]; then
   WGET_URL1=http://vision.maths.lth.se/calledataset/fine_arts_palace/fine_arts_palace.zip
   WGET_URL2=http://vision.maths.lth.se/calledataset/fine_arts_palace/data.mat
   echo $WGET_URL1
   echo $WGET_URL2
```

Comment:
unrelated: remove the newlines for these and move the echo statements outside the elif? If we really need them, otherwise can remove them too.

Reply:
Should I remove all of the newlines in the other if else statement? I think it looks cleaner but i can remove them.

Moved the echo as I think it's still useful to have it in case of wget failures.